### PR TITLE
Polls refinements

### PR DESCRIPTION
**Pull request by alfogrillo**
This PR:
- Deletes the feature flags for polls
- Updates the "end poll" UI in the timeline
- Refines the focus logics in the "create poll" form
- Adds an alert when the user discards a "draft poll"

### CODE REVIEW

File: ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift [hunk 1]
```diff
@@ -39,6 +39,8 @@ struct CreatePollScreen: View {
         .navigationBarTitleDisplayMode(.inline)
         .toolbar { toolbar }
         .animation(.elementDefault, value: context.options)
+        .interactiveDismissDisabled(context.viewState.bindings.hasDraftContent)
```
Comment (stefanceriu) [inline ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift:42]: Shouldn't this be always disabled? I don't want to lose even my draft content by mistake 🤔

File: UITests/Sources/CreatePollScreenUITests.swift [hunk 2]
```diff
@@ -56,6 +56,9 @@ class CreatePollScreenUITests: XCTestCase {
             addOption.tap()
         }
 
+        if app.keyboards.count > 0 {
```
Comment (stefanceriu) [inline UITests/Sources/CreatePollScreenUITests.swift:59]: 🙊

**Review by stefanceriu — APPROVED** [review]
lgtm 👍

File: ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift (hunk 1 above)
Comment (alfogrillo) [reply ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift:42]: The idea is to disable the interaction if the user wrote at least some character somewhere. 
If the screen is completely empty we still give the possibility to drag down the sheet. Does it make sense?
Maybe just `hasContent` clarifies better the intent?

File: UITests/Sources/CreatePollScreenUITests.swift (hunk 2 above)
Comment (alfogrillo) [reply UITests/Sources/CreatePollScreenUITests.swift:59]: Playing with `@FocusState` apparently doesn't work like expected on iPad.
But I fell like is a minor problem.

Comment (stefanceriu) [reply UITests/Sources/CreatePollScreenUITests.swift:59]: Yeah, don't worry about it too much

File: ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift (hunk 1 above)
Comment (alfogrillo) [reply ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift]: Applied!
https://github.com/vector-im/element-x-ios/pull/1608/commits/04e63f5d0dd2781ba8944955e966ead71b138ecc

Comment (stefanceriu) [reply ElementX/Sources/Screens/CreatePollScreen/View/CreatePollScreen.swift]: Thanks!

**Pull request merged**